fosdem/stand-cheatsheet.adoc

```diff
@@ -0,0 +1,102 @@
+* *Pipeline:* A couple of example jobs using the link:https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin[Pipeline plugin] to implement a few projects
+* *Python:* A couple examples jobs using the plugins suggested on link:https://jenkins-ci.org/s/python[jenkins-ci.org/s/python]
+* *Jenkins 2.0:* A locally running Jenkins 2.0 build on the Mac itself (not containerized)
+
+There is of course always link:https://ci.jenkins-ci.org[ci.jenkins-ci.org] which is a good real-live demo of Jenkins to talk about.
+
+==== Running demos
+
+- [ ] Click the "Docker Quickstart Terminal" in the Dock
+- [ ] In the terminal that comes up, execute: `cd demos && ./run_demos`
+- [ ] Navigate to one of the demo Jenkins instances bookmarked in Chrome, e.g. "Demo - PHP"
+
+
+Running the "Jenkins 2.0" demo, which only really can be used to show off the
+setup wizard, can be done with the `demos/run_jenkins2` script, which will
+create a fresh `JENKINS_HOME` for each execution.
+
+=== Logging in
+
+The primary user account is named "mrjenkins" and the password for this user is
+"jenkins!".
+
+=== Troubleshooting
+
+==== Networking is broken in Docker containers
+
+`boot2docker` doesn't appear to switch networks properly on Mac OS X. If
+Jenkins running inside a Docker container cannot reach the internet follow
+these steps:
+
+- [ ] Can the Mac reach the internet in Chrome; if it cannot, the fix that
+  first.
+- [ ] Stop all containers (using the `stop_all_containers` script in the
+  `demos/` directory
+- [ ] Open VirtualBox application (in the Dock)
+- [ ] Power off the `default` virtual machine (*⌘F*); quit VirtualBox
+- [ ] Close all terminals and re-open "Docker Quickstart Terminal (in the
+  Docker). This will properly restart the intermediate virtual machine.
+- [ ] Restart the demo containers with: `cd demos && ./run_demos`
+
+
+== Swag
+
+=== Stickers
+
+We have a number of different kinds of stickers, the names are:
+
+* Jenkins (traditional logo)
+* Angry Jenkins (fiery red logo)
+* Ninja Jenkins
+* General Jenkins (Jenkins in a military outfit)
+* Cute Jenkins (Jenkins making a Anime-style "whoops" face)
+* Superhero Jenkins
+
+=== Pins
+
+We have two kinds of pins for general give-away:
+
+. Jenkins on blue background
+. Angry Jenkins on red background
+
+The pins with flags on them are for contributors and JAM organizers.
+
+=== Postcard
+
+The postcard printed should be given to everybody that receives or takes a
+sticker and/or pin.
+
+On the front of the postcard are quick links to "solution pages" which contain
+links to plugins, articles, presentations and other documentation for using
+Jenkins on that platform/toolchain.
+
+On the back of the postcard is a current map for regional Jenkins Area
+Meetups,a live version of this map can be shown/found at
+link:https://www.meetup.com/pro/jenkins[meetup.com/pro/jenkins].
+
+
+== Merchandise
+
+=== T-Shirts
+
+T-shirts are *not free* but are to be sold, all proceeds go directly to the
```

Comment:
Do we know the price they should be sold for?  I guess we can print / write that on separate signs anyway.